matrix-meetings-widget/src/components/meetings/MemberSelectionDropdown/MemberSelectionDropdown.tsx

```diff
@@ -394,16 +502,24 @@ export function MemberSelectionDropdown({
       <Autocomplete
         disableCloseOnSelect
         disablePortal
-        filterSelectedOptions
+        filterSelectedOptions={!searchHomeserverUsers}
         fullWidth
         getOptionLabel={getOptionLabel}
+        filterOptions={searchHomeserverUsers ? filterOptions : undefined}
         id={id}
         multiple
-        noOptionsText={t(
-          'memberSelectionDropdown.noMembers',
-          'No further members.'
-        )}
+        loading={searchHomeserverUsers ? loading : undefined}
+        noOptionsText={
+          searchHomeserverUsers && term.length === 0
+            ? t(
+                'memberSelectionDropdown.typeToSearch',
+                'Type to search for a user…'
+              )
+            : t('memberSelectionDropdown.noMembers', 'No further members.')
+        }
+        loadingText={t('memberSelectionDropdown.loading', 'Loading…')}
         onChange={handleOnChange}
+        onInputChange={searchHomeserverUsers ? handleInputChange : undefined}
```

Comment:
Why have this conditional? I think this was in order to not trigger the loading of the search results, but when we can explicitly disable the hook, I guess this shouldn't be necessary.

Reply:
changed this, not really needed anymore